### PR TITLE
Bulgarian Translation 17

### PR DESCRIPTION
**Pull request by Effektus**
99.9% Finished

### CODE REVIEW

**Comment by Effektus** [comment]
[zenseii](https://github.com/zenseii) Ivan please check fuzzy marks. I have difficult to translate this one

File: files/lang/bg.po [hunk 1]
```diff
@@ -9008,55 +9059,74 @@ msgid ""
...
+"При изкачване на малък хълм се натъкваш на нелепо изглеждаща гледка. Фея се "
+"опитва да носи скиптър, който е почти толкова голям, колкото нея. Опитвайки "
+"се да не се смееш, ти питаш: \"Имаш ли нужда от помощ?\" Феята ви гледа "
+"ядосано и отговаря: \"Мислиш, че това е смешно? Добре. Може да го носиш. И "
+"без това предпочитам да летя.“"
 
+#, fuzzy
 msgid "Masthead"
-msgstr ""
+msgstr "Заглавие"
```
Comment (zenseii) [inline files/lang/bg.po:9070]: This means the top of a mast that you have on sail boats.
This is what the artifact looks like:
image

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi @Effektus.

I've left a comment about the fuzzied string. I'll review the rest of the translation later.

File: files/lang/bg.po (hunk 1 above)
Comment (Effektus) [reply files/lang/bg.po:9070]: Is this right? 
image

Comment (zenseii) [reply files/lang/bg.po:9070]: Yep, in Heroes 2 the orcs look a lot like pigs. Not sure if that's what you're asking.

File: files/lang/bg.po [hunk 2]
```diff
@@ -9008,55 +9059,74 @@ msgid ""
 "answers: \"You think this is funny? Fine. You can carry it. I much prefer "
 "flying anyway.\""
 msgstr ""
+"При изкачване на малък хълм се натъкваш на нелепо изглеждаща гледка. Фея се "
+"опитва да носи скиптър, който е почти толкова голям, колкото нея. Опитвайки "
+"се да не се смееш, ти питаш: \"Имаш ли нужда от помощ?\" Феята ви гледа "
+"ядосано и отговаря: \"Мислиш, че това е смешно? Добре. Може да го носиш. И "
+"без това предпочитам да летя.“"
```
Comment (zenseii) [inline files/lang/bg.po:9066]: Please use the other quotation mark.
```suggestion
"без това предпочитам да летя.\""
```

File: files/lang/bg.po [hunk 3]
```diff
@@ -9112,446 +9189,498 @@ msgid ""
...
-msgstr ""
+msgstr "Кара земните елементали да пазят мина срещу вражески армии."
 
 msgid "Set Air Guardian"
-msgstr ""
+msgstr "Задай Въздушен паззител"
 
 msgid "Sets Air Elementals to guard a mine against enemy armies."
-msgstr ""
+msgstr "Кара Въздушни елемнетали да пазят мина срещу вражески армии."
```
Comment (zenseii) [inline files/lang/bg.po:9652]: Small typo
```suggestion
msgstr "Кара Въздушни елементали да пазят мина срещу вражески армии."
```

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi, @Effektus.

I left a couple of comments. Please have a look.

File: files/lang/bg.po (hunk 3 above)
Comment (Effektus) [reply files/lang/bg.po:9652]: What is the difference here?

Comment (zenseii) [reply files/lang/bg.po:9652]: не and ен
`елемнетали`
`елементали`

**Review by zenseii — CHANGES_REQUESTED** [review]

**Review by zenseii — APPROVED** [review]

**Review by zenseii — APPROVED** [review]

File: files/lang/bg.po [hunk 4]
```diff
@@ -6658,85 +6658,85 @@ msgid "speed|Instant"
...
-msgstr "Пеперуда"
+msgstr "Пеперудата"
 
 msgid "week|Bumblebee"
-msgstr "Земна пчела"
+msgstr "Земната пчела"
 
 msgid "week|Locust"
-msgstr "Скакалец"
+msgstr "Ненаситника"
```
Comment (zenseii) [inline files/lang/bg.po:6724]: Is there a specific reason this was changed? I can't seem to find any reference showing that Ненаситника refers to a grasshopper/locust in Bulgarian. It seems to mean "insatiable"?

**Review by zenseii — COMMENTED** [review]
Hi, @Effektus.

I left one comment. Please have a look at it.

**Review by zenseii — APPROVED** [review]

**Review by ihhub — CHANGES_REQUESTED** [review]

File: files/lang/bg.po [hunk 5]
```diff
@@ -306,7 +306,7 @@ msgstr "%{name} се предава на врага и си тръгва зас
 
 msgid "Your force suffer a bitter defeat, and %{name} abandons your cause."
 msgstr ""
-"Твойте сили претърпяват горчиво поражение и %{name} изоставя каузата ви."
+"Твойте сили претърпяват горчиво поражение и %{name} изоставя каузата ти."
```
[the remark this answers is not in the document]
Comment (Effektus) [reply files/lang/bg.po:339]: Because in Bulgarian the official form "ви" is used quite often and for the plural.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Effektus , thanks a lot for this update!

**Pull request merged**